bucket/scoop-search.json

```
@@ -0,0 +1,13 @@
+{
+    "version": "0.6.5",
+    "license": "MIT",
+    "url": "https://github.com/tokiedokie/scoop-search/releases/download/v0.6.5/scoop-search.exe",
+    "homepage": "https://github.com/tokiedokie/scoop-search",
+    "description": "yet another scoop search",
+    "hash": "9f180302f497cfcf371053a4ff151c230cdf62baebc2007da19ee0749d9b3fcf",
+    "bin": "scoop-search.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/tokiedokie/scoop-search/releases/download/v$version/scoop-search.exe"
+    }
+}
```

```suggestion
}

```

Insert final newline for tests to pass